google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/TransactionRunnerImpl.java

```diff
@@ -270,7 +271,12 @@ public long executeUpdate(Statement statement) {
         results[i] = response.getResultSets(i).getStats().getRowCountExact();
       }
 
-      if (response.getStatus().getCode() != 0) {
+      // If one of the DML statements was aborted, we should throw an aborted exception.
```

Comment:
is it possible to also add a test to TransactionRunnerImplTest?

Reply:
Added.